packages/orbit-components/src/Loading/Loading.stories.tsx

```diff
@@ -1,123 +1,113 @@
-        <CardSection>
-          <Illustration name="EnjoyApp" size="medium" />
-        </CardSection>
-      </Loading>
-    </Card>
-  );
-};
-
-CardLoading.story = {
-  name: "Card loading",
-
-  parameters: {
-    info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+  argTypes: {
+    type: {
+      options: Object.values(TYPE_OPTIONS),
+      control: {
+        type: "select",
+      },
+    },
   },
 };
 
-CardLoading.args = {
-  title: "Card with title",
-  description: "",
-  loading: true,
-  loadingText: "Please wait, Card content is loading...",
-};
+export default meta;
 
-export const InlineLoader = ({ loadingText }) => {
-  return <Loading type={TYPE_OPTIONS.INLINE_LOADER} text={loadingText} />;
-};
+type Story = StoryObj<typeof Loading>;
 
-InlineLoader.story = {
-  name: "Inline loader",
+export const Default: Story = {
+  args: {
+    customSize: undefined,
+    loading: undefined,
+    text: undefined,
+    type: undefined,
+  },
 
   parameters: {
-    info: "This configuration of this component is without any height, width or paddings. Visit Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      disable: true,
+    },
   },
 };
 
-InlineLoader.args = {
-  loadingText: "Please wait, content of the page is loading...",
-};
+export const DefaultWithChildren: Story = {
+  render: args => (
+    <Loading {...args}>
+      <Text>Loading prop is false.</Text>
+    </Loading>
+  ),
 
-export const CustomLoader = ({ customSize }) => {
-  return <Loading customSize={customSize} />;
-};
+  parameters: {
+    controls: {
+      exclude: ["children", "customSize", "text", "type"],
+    },
+  },
 
-CustomLoader.args = {
-  customSize: 50,
+  args: {
+    customSize: undefined,
+    text: undefined,
+    type: undefined,
+  },
 };
 
-export const Playground = ({ type, loadingText, loading, dataTest }) => {
-  return <Loading loading={loading} type={type} text={loadingText} dataTest={dataTest} />;
-};
+export const Playground: Story = {
+  render: ({ customSize, ...args }) => (
+    <div className="stroke-blue-dark">
+      <Loading {...args} customSize={customSize === 0 ? undefined : customSize} />
+    </div>
+  ),
 
-Playground.story = {
-  parameters: {
-    info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+  args: {
+    loading: undefined,
   },
-};
-
-Playground.args = {
-  type: TYPE_OPTIONS.PAGE_LOADER,
-  loadingText: "Please wait, content of the page is loading...",
-  loading: true,
-  dataTest: "test",
-};
 
-Playground.argTypes = {
-  type: {
-    options: Object.values(TYPE_OPTIONS),
-    control: {
-      type: "select",
+  parameters: {
+    info: "Playground of Loading component. Type buttonLoader inherits parent's stroke color. For visualisation of undefined value for customSize arg, set value number 0. Visit Orbit.Kiwi for more detailed guidelines.",
```

Comment:
Some of the component props depend on the specific configuration, so I wrote some extended instructions and explanations.